public/js/audioseeker.js

```diff
@@ -10,3 +10,73 @@ player.addEventListener("timeupdate", function () {
       "linear"
     );
 });
+
+function sayLoc(e) {
```

Comment:
we can move this whole file to `utils` folder and directly import and use in `AudioPlayer.js`

Reply:
Same as Previous

Reply:
Please can you specify how because some of the functions are handling events and I need it to be a javascript file being rendered  in the frontend